### DISCUSSION TITLE
Word Documents Created by Pandoc

### DISCUSSION THREAD

**Discussion by tom-mclean** [retorquere/zotero-better-bibtex#3071] [Q&A]
I didn't know I was using BetterBibTeX to do this as I'm certain I only clicked on things in the Zotero menu bar in Word, but I've been directed here from the Zotero forums! Can anyone help? What I asked there is as follows:

Before I upgraded to Zotero 7, I could create Word documents using Pandoc that contained references inline like "[Some text about why I am citing this source, [@authornameDATE, p. xx]" and when I opened the Word document, once I had identified a citation style in the Zotero tool bar, it would scan the document and put the citations in properly (where [@authornameDATE] is the BetterBibTeX citation key saved in Zotero). Occasionally this would glitch with footnote based systems, but if I used an inline author-date style first and then changed the style once Zotero had scanned the text, it was fine (and everything in the brackets ended up as a footnote).

Since I've upgraded, this doesn't seem to work anymore. Is this because that feature is no longer included, or because I need to do something else to make it work?

I would rather not convert to RTF and use the RTF scanner within the Zotero app proper, to avoid losing formatting elsewhere in the document.

(If it makes a difference, Zotero 7.0.9 on a Mac. Both Zotero and Word report being up to date. BetterBibTeX is also still installed and up to date).

Thanks!

**Comment by retorquere** [comment]
Did the discussion on the zotero forums cover this problem?

**Reply by tom-mclean** [reply]
Yes, covered, thank you!
